CLI/dsi2lsl.c

```diff
@@ -1,20 +1,30 @@
 /*
-This file implements the integration between Wearable Sensing DSI C/C++ API and
-the LSL library.
-
-Copyright (C) 2014-2020 Syntrogi Inc dba Intheon.
+* This file implements the integration between Wearable Sensing DSI C/C++ API and
+* the LSL library.
+*
+* This program is used for acquiring data from a DSI headset and streaming it
+* over the Lab Streaming Layer (LSL) protocol, which allows for real-time
+* data acquisition and analysis.
+
+* This program utilizes windows threads to handle the DSI headset processing
+* and impedance checking in parallel, allowing for real-time data acquisition.
+* DSI headset processing threads continuously call DSI_Headset_Idle to process
+* incoming data, while the impedance thread checks for impedance activity and
+* prints results.
+*
+* Copyright (C) 2014-2020 Syntrogi Inc dba Intheon.
```

Comment:
You can remove the copyright

Reply:
add instead: 

Please create a GitHub Issue or contact support@wearablesensing.com if you encounter any issues or would like to request new features.